build/build.sh

```diff
@@ -14,9 +14,9 @@ Options:
 	-help										Display this help
 	-clean									Clean build directory
 	-debug									Only debug build
-	-release								release build
-	-example								example build
-	-package								Generate xrt_plugin .deb package
+	-release								release build and generate .deb package
+	-example								example build and generate .deb package
```

Comment:
It seems like "-example" is not changed. But "-debug" is changed to also build package?